ahk/utils.py

```diff
@@ -1,6 +1,6 @@
 import logging
 import functools
-from asyncio import coroutine
+from asyncio import coroutines
```

Comment:
It seems this import is unused, anyhow. I think it should be safe to remove altogether.

```suggestion
```